src/templates/app/cashier/deposit_withdraw.jsx

```diff
@@ -18,7 +18,7 @@ const DepositWithdraw = () => (
                 {it.L('There was a problem validating your personal details. Please update your [_1] <a href="[_2]">here</a>.', '[_1]', it.url_for('user/settings/detailsws'))}
                 <br /> {it.L('If you need assistance feel free to contact our <a href="[_1]">Customer Support</a>.', it.url_for('contact'))}
             </p>
-            <p id='not_authenticated_message' />
+            <a id='not_authenticated_message' href={it.url_for('user/authenticate')} />
```

Comment:
this means that the whole message will become a link. it's better to only highlight `authenticate` word. my suggestion is that instead of showing the back-end error message which stops us from linking just one word, use a front-end error message instead